performance-evaluation/google_drive.py

```diff
@@ -0,0 +1,53 @@
+from constants import GDRIVE_FILE_NAME
+from pydrive.auth import GoogleAuth
+from pydrive.drive import GoogleDrive
+from percol import Percol
+import readchar
+
+
+def select_from_list(folders):
+    si, so, se = "", "", ""
+    candd = [p[0] for p in folders]
```

Comment:
I guess `candidates` is a better variable name here. And what's `p` in this context?

Reply:
Done